arangod/Cluster/ClusterInfo.cpp

```diff
@@ -2100,13 +2100,14 @@ Result ClusterInfo::dropCollectionCoordinator(  // drop collection
 
   if (!clones.empty()) {
     std::string errorMsg(
-        "Collection must not be dropped while it is a sharding prototype for "
+        "Collection must not be dropped while "
         "collection(s)");
 
     for (auto const& i : clones) {
       errorMsg += std::string(" ") + i;
```

Comment:
While we're at it, should we maybe add a comma separation or such? Maybe restructure the message to `"Collection must not be dropped while used as distributeShardsLike target for collection(s): " + StringUtils::join(....)`?